sdk/eventhub/event-hubs/test/receiver.spec.ts

```diff
@@ -99,27 +94,24 @@ describe("EventHub Receiver", function(): void {
           stamp: uid
         }
       };
+
+      const offset = (await client.getPartitionInformation(partitionId)).lastEnqueuedOffset;
```

Comment:
This particular test is for `from end of stream` i.e testing of the event position as returned by `EventPosition.fromEnd()`. We have now updated it to use the `lastEnqueuedOffset` instead which makes this test identical to the one for `after a particular offset`

I don't see a way out for this case. Let's comment this test out and add a TODO comment to bring it back as part of https://github.com/Azure/azure-sdk-for-js/issues/3714

Reply:
Oh, there might have been a mistake, thanks for the catch!
This should be updated to `EventPosition.fromEnd()` and expect to receive 0 messages.

Reply:
Updated

Reply:
The intent was not to check that receiving from "end of stream" receives 0 messages. The intent was that a receiver created using the `EventPosition.fromEnd()`, will receive the messages that were sent after creating the receiver.

This is not possible without having the initial `receive()` call because the link is established only when the `receive()` call is made.

Therefore, my suggestion in https://github.com/Azure/azure-sdk-for-js/pull/3709#discussion_r293192981 was to undo the changes to this test and comment it out

Reply:
> The intent was not to check that receiving from "end of stream" receives 0 messages. The intent was that a receiver created using the `EventPosition.fromEnd()`, will receive the messages that were sent after creating the receiver.
> 
> This is not possible without having the initial `receive()` call because the link is established only when the `receive()` call is made.
> 
> Therefore, my suggestion in [#3709 (comment)](https://github.com/Azure/azure-sdk-for-js/pull/3709#discussion_r293192981) was to undo the changes to this test and comment it out

Oh, okay - will undo and comment out.

Reply:
Agreed with the basic happy path, but since the intent of the test was more than just the happy path, I would prefer to comment it out.

The main intention of this exercise/PR/issue is to fix the flakiness. As part of the Track 2 work, we will look into increasing test coverage where we will revisit tests and maybe add this happy path.